hazen.py

```diff
@@ -11,37 +11,64 @@
 __author__ = "mohammad_haris.shuaib@kcl.ac.uk"
 
 
+import os
+import inspect
+
+import os
+import inspect
+import ast
+
+
 def register_tasks_in_db():
     from hazenlib import tasks as hazen_tasks
-    # TODO: Change to read from hazenlib classes
-    tasks = {f'{modname}': importlib.import_module(
-        f'hazenlib.tasks.{modname}') for importer, modname, ispkg in
-        pkgutil.iter_modules(hazen_tasks.__path__)}
+    tasks = {
+        f'{modname}': importlib.import_module(f'hazenlib.tasks.{modname}')
+        for importer, modname, ispkg in pkgutil.iter_modules(hazen_tasks.__path__)
+    }
+
+    def get_module_docstring(filepath):
```

Comment:
If we keep this function in e.g., not use a module's `__doc__` global, I suggest we move this outside the `register_tasks_in_db` function.